src/test/java/com/fauna/beans/ClassWithParameterizedFields.java

```diff
@@ -17,4 +18,24 @@ public class ClassWithParameterizedFields {
 
     @FaunaField(name = "a_map", typeArgument = Integer.class)
     public Map<String,Integer> map  = Map.of("key1", 42);
+
+    @Override
+    public boolean equals(Object o) {
+        if (this == o) return true;
+
```

Comment:
Is it required that users override the `equals` method like this for all of their classes?

Reply:
Nope. This just makes test comparisons easier.